src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -808,7 +808,9 @@ bool PolicyHandler::IsAppSuitableForPolicyUpdate(
     const Applications::value_type value) const {
   LOG4CXX_AUTO_TRACE(logger_);
 
+#ifdef ENABLE_LOG
   const uint32_t app_id = value->app_id();
+#endif
```

Comment:
Probably better just to remove this variable than to use a macro

Reply:
OK, removed by https://github.com/smartdevicelink/sdl_core/pull/1789/commits/1b8e82a38e879a022647e6e17498c9865d7853e3.